server/webapp/WEB-INF/rails.new/webpack/views/dashboard/edit_tab_dropdown.js.msx

```diff
@@ -42,7 +42,11 @@ const EditTabDropdown = {
       const current = vm.canonicalCurrentName();
       const allowDelete = !vm.locked() && !vm.isDefault(current);
 
-      const edit = vm.actionHandler(() => Editor.open(model().namedFilter(current), model, vm));
+      const edit = vm.actionHandler(() => {
```

Comment:
I think I generally agree, but I have a concern. Mainly, I'm concerned about time. We're getting pretty close to the release, and I think this change wouldn't be a trivial amount of work.

So, I'm wondering if this can be a separate PR that I can start work on now?

Reply:
@ibnc we could time box it. While it won't be a 1-line change, I think it won't be terribly difficult either.

While the stream is unset, we could display a spinner for the list. after the ajax call returns, flip it back. This type of thing could be easily baked into the `view()` function of the component.

Example:

```
const SelectedPipelineList = {
  view(vnode) {
    const vm = vnode.attrs.vm;
    if (vm.displayedList()) { // alternatively you could just have a `loading()` flag
      return <div class="loading"><i class="spinner"/></div>;
    }

    const groups = vm.displayedList();
    return m("ul", {class: "selected-pipelines"},
      _.map(groups, (groupVM, name) => {
        return <PipelineGroupSelection vm={groupVM} name={name} />;
      })
    );
  }
};
```

And `vm.displayedList()` returns `null` or something when it hasn't loaded anything.